plugins/webapi/tips.go

```diff
@@ -113,49 +113,16 @@ func getTipInfo(i interface{}, c *gin.Context, _ <-chan struct{}) {
 		return
 	}
 
-	cachedBundle := tangle.GetCachedBundleOrNil(cachedTxMeta.GetMetadata().GetTxHash()) // bundle +1
-	if cachedBundle == nil {
-		e.Error = "unknown bundle"
-		c.JSON(http.StatusBadRequest, e)
+	// if the OTRSI to LSMI delta is over MaxDeltaTxOldestRootSnapshotIndexToLSMI, the tip is semi-lazy and should be promoted
+	if (lsmi - ortsi) > milestone.Index(config.NodeConfig.GetInt(config.CfgTipSelMaxDeltaTxOldestRootSnapshotIndexToLSMI)) {
+		c.JSON(http.StatusOK, GetTipInfoReturn{
+			Confirmed:      false,
+			Conflicting:    false,
+			ShouldPromote:  true,
```

Comment:
If a tip is semi-lazy it can be promoted?
I thought it couldn't because the ORTSI doesn't change

Reply:
The promotion tx will also land in the semi-lazy pool, but the chance slightly increases that it will be picked.